packages/@aws-cdk/aws-neptune/lib/cluster.ts

```diff
@@ -113,13 +96,6 @@ export interface DatabaseClusterProps {
    */
   readonly associatedRoles?: iam.IRole[];
 
-  /**
-   * Indicates whether the DB cluster is IAM auth enabled.
-   *
-   * @default - false, connections to database don't require IAM auth
-   */
-  readonly iamAuthEnabled?: boolean;
```

Comment:
Why was this removed?

Reply:
I wanted to test this before adding support for it, found some conflicting information where AWS CLI says IAM authentication is not supported by Neptune at the cluster level but is supported at instance level (whereas CFN has no flag to pass this value at instance level). Wanted to check if SDK documentation is incorrect or if it is a really an instance specific setting and if so whether CFN support for IAM needs to be fixed.